### DISCUSSION TITLE
Star exchange

### DISCUSSION THREAD

**Discussion by ternera** [Schweinepriester/achievement-testarea#46] [General]
Please star and I will return it: https://github.com/ternera/ternera.org

**Comment by AdityaNow** [comment]
Done 😃 (feel free to star my repo https://github.com/AdityaNow/NavtorAssignment )
Thank you

**Reply by KamBRJC** [reply]
give you a star, please give me back https://github.com/z-nnovation/GitSwitcher

**Reply by AdityaNow** [reply]
> give you a star, please give me back https://github.com/z-nnovation/GitSwitcher

Done 😃

**Comment by peterbaccaro** [comment]
Please give me a star for my repo and help me get ⭐️ starstruck ⭐️!

https://github.com/peterbaccaro/design-patterns-python

Much appreciated. Thanks in advance.

**Reply by KamBRJC** [reply]
back me star please https://github.com/z-nnovation/GitSwitcher

**Comment by matteofuso** [comment]
Done, please star (https://github.com/matteofuso/musicdownloader)

Thanks

**Reply by AdityaNow** [reply]
Done 😃 (please star my repo https://github.com/AdityaNow/NavtorAssignment )
Thank you

**Reply by KamBRJC** [reply]
star my https://github.com/z-nnovation/GitSwitcher

**Comment by Denisyasyuchenya** [comment]
give you a star

star my https://github.com/Denisyasyuchenya/my-location-app

**Reply by AdityaNow** [reply]
Done 😃 (please star my repo https://github.com/AdityaNow/NavtorAssignment )
Thank you

**Comment by KamBRJC** [comment]
star back https://github.com/z-nnovation/GitSwitcher

**Reply by AdityaNow** [reply]
thank you 😃

**Comment by Vlada3333** [comment]
There is a star!
